README.rst

```diff
@@ -615,6 +615,11 @@ Choose from the list of available fixers:
                         closing semicolon are
                         prohibited.
 
+* **native_function_casing** [contrib]
```

Comment:
it is not risky, you can change the casing in the example of both the call and the function name in any random way and it would not effect the code behavior